Makefile

```diff
@@ -171,13 +175,13 @@ docker-push:
 docs: ## Regenerates flags in docs for all thanos commands localise links, ensure GitHub format.
 docs: $(MDOX) build
 	@echo ">> generating docs"
-	PATH=${PATH}:$(GOBIN) $(MDOX) fmt --links.localize.address-regex="https://thanos.io/.*" $(MD_FILES_TO_FORMAT)
+	PATH=${PATH}:$(GOBIN) $(MDOX) fmt -l --links.localize.address-regex="https://thanos.io/.*" --links.validate.config-file=$(MDOX_VALIDATE_CONFIG) $(MD_FILES_TO_FORMAT)
```

Comment:
Can we double check the right value for `--links.localize.address-regex="https://thanos.io/.*`? 

e.g this link was not localised: https://deploy-preview-4357--thanos-io.netlify.app/tip/proposals-accepted/202005-scalable-rule-storage.md/

![image](https://user-images.githubusercontent.com/6950331/123232641-43761e80-d4d1-11eb-8487-ac3374e77943.png)

Reply:
I've changed this to a local link.